src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -314,7 +314,7 @@ def get_latest_from_github(package_path='azure-cli'):
 
 
 def _update_latest_from_github(versions):
-    if not check_connectivity(max_retries=0):
+    if not check_connectivity(url='https://raw.githubusercontent.com', max_retries=0):
```

Comment:
The default URL is azure. We'd better use `raw.githubusercontent.com` to check GitHub connection status.

Reply:
1. Why does using `raw.githubusercontent.com` fails faster?
2. Also, why do we even need this check? Why don't we just check connectivity and call `get_latest_from_github(pkg)` in one shot?

Reply:
1. CLI opens `https://raw.githubusercontent.com/Azure/azure-cli/main/src/{}/setup.py` two times to get version info, the timeout of each openration is 10s.  `check_connectivity` uses `HEAD` to test connection, its timeout is 1s. If `check_connectivity` fails, it skips the url open process.
3. `check_connectivity` can be removed. But it saves time when github.com is not reachable. 1s vs 2*10s.